modules/docs/mdx/10.0-UPGRADE_GUIDE.mdx

```diff
@@ -24,34 +24,7 @@ any questions.
 
 ## Codemod
 
-Please use our [codemod package](https://github.com/Workday/canvas-kit/tree/master/modules/codemod)
-to automatically update your code to work with most of the breaking changes in v10.
-
-```sh
-> npx @workday/canvas-kit-codemod v10 [path]
-```
-
-Alternatively, if you're unable to run the codemod successfully using `npx`, you can install the
-codemod package as a dev dependency, run it with `yarn`, and then remove the package after you're
-finished.
-
-```sh
-> yarn add @workday/canvas-kit-codemod --dev
-> yarn canvas-kit-codemod v10 [path]
-> yarn remove @workday/canvas-kit-codemod
-```
-
-> The codemod only works on `.js`, `.jsx`, `.ts`, and `.tsx` files. You'll need to manually edit
-> other file types (`.json`, `.mdx`, `.md`, etc.). You may need to run your linter after executing
-> the codemod, as its resulting formatting (spacing, quotes, etc.) may not match your project
-> conventions.
-
-The codemod will handle _most_ but _not all_ of the breaking changes in 10. **Breaking changes
-handled by the codemod are marked with 🤖 in the Upgrade Guide.**
-
-**Please verify all changes made by the codemod.** As a safety precaution, we recommend committing
-the changes from the codemod as a single isolated commit (separate from other changes) so you can
-roll back more easily if necessary.
+Unlike past major releases, v10 does not have a codemod.
```

Comment:
Would be nice to leave more explanation here about why there's no codemod (i.e., it didn't make sense for v10, but codemods aren't going away in the future). Open to suggestions.

Reply:
+1 yeah "There is no codemod for v10 since most changes don't require a rewrite of the code. However, this does not mean there won't be in the future. We hope this upgrade is sufficient enough to cover changes in this major version." idk

Reply:
I ended up citing infeasibility and ROI to explain the lack of a codemod. To your comment, I included a bit about the Upgrade Guide providing enough instruction to update your code without needing a codemod.